examples/app/config/administrator/settings/site.php

```diff
@@ -32,7 +32,7 @@
 			'title' => 'Image (200 x 150)',
 			'type' => 'image',
 			'naming' => 'random',
-			'location' => public_path(),
+			'location' => public_path().'/',
```

Comment:
this is not a change we will make if you can take this out we can consider the options_filter part.